http/src/routing.rs

```diff
@@ -367,6 +367,17 @@ pub enum Route<'a> {
         /// The ID of the guild.
         guild_id: u64,
     },
+    /// Route information to get a guild's bans with parameters.
+    GetBansWithParameters {
+        /// User ID after which to retrieve bans.
+        after: Option<u64>,
+        /// User ID before which to retrieve bans.
+        before: Option<u64>,
+        /// Maximum number of bans to retrieve.
+        limit: Option<u16>,
+        /// The ID of the guild.
```

Comment:
A single the should be enough.

```suggestion
        /// ID of the guild.
```